ci/_environment.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-ANDROID_BUILDER_TAG=c0de63a4ce
+ANDROID_BUILDER_TAG=95492bd0b2
```

Comment:
What has been changed?
Don't see changes in the image.

Reply:
nothing, just rebuild image; sdk revision changed underhood (no versioning, known issue)

Reply:
Why this change in this PR?

Reply:
Will you update an image in the internal repository?